src/Sarif/Baseline/ResultMatching/DataStructures/MatchedResults.cs

```diff
@@ -228,22 +228,6 @@ internal void SetFirstDetectionTime(Result targetResult)
             targetResult.Provenance.FirstDetectionTimeUtc = firstDetectionTime;
         }
 
-        private Dictionary<string, object> MergeDictionaryPreferFirst(
```

Comment:
>MergeDictionaryPreferFirst [](start = 43, length = 26)

Moved to ExtensionMethods.cs because the `GitHubDspIngestionVisitor` now needs it, too.